cfme/services/myservice/ssui.py

```diff
@@ -225,6 +241,20 @@ def launch_vm_console(self, catalog_item):
     return vm_obj
 
 
+@MyService.retire.external_implementation_for(ViaSSUI)
+def retire(self):
+    view = navigate_to(self, 'Retire')
+    view.retire.click()
+    view = self.create_view(MyServicesView)
+    wait_for(
+        lambda: view.is_displayed, delay=15, num_sec=300,
+        message="waiting for view to be displayed"
+    )
+    assert view.is_displayed
```

Comment:
**Optional**

Consider moving this assert to L249, and assert the return from wait_for, since it'll be the result of the last call to `view.is_displayed`.
https://github.com/RedHatQE/wait_for/blob/master/wait_for/__init__.py#L116

Reply:
assert for view.is_displayed cannot be moved before wait as it fails without wait.

Reply:
No, I mean that wait_for returns what view.is_displayed returned - so you can just assert against that wait for statement. Not put the call before, make it one line.

Reply:
done